pkg/reporter/reporter_test.go

```diff
@@ -13,9 +13,17 @@ func TestBuildUrl(t *testing.T) {
 		Password: "foobar",
 	}
 
+	Version = 1
```

Comment:
even though the global exists, I'd try to rework the function so that no global state needs to be modified for testing (where version is passed into the `buildURL` function). This way it would be the caller (or another function wrapper) thats responsible for the maintenance of the global (and the global gets decoupled from the business logic).

Reply:
Done!